roles/test_operator/README.md

```diff
@@ -65,11 +78,11 @@ Execute tests via the [test-operator](https://openstack-k8s-operators.github.io/
         {{ cifmw_test_operator_tempest_include_list | default('') }}
       excludeList: |
         {{ cifmw_test_operator_tempest_exclude_list | default('') }}
```

Comment:
**issue (blocking):** both `cifmw_test_operator_tempest_tests_include_override_scenario` and `cifmw_test_operator_tempest_tests_exclude_override_scenario` do not seem to follow the agreed upon precedence of `test_vars` > `test_vars_file` > `global_vars`

When I use this configuration:
```
cifmw_test_operator_tempest_tests_include_override_scenario: true
cifmw_test_operator_tempest_tests_exclude_override_scenario: true

cifmw_test_operator_stages:
  - name: tempest
    type: tempest
    test_vars:
      cifmw_test_operator_tempest_tests_include_override_scenario: false
      cifmw_test_operator_tempest_tests_exclude_override_scenario: false
```

then these two blocks of ansible tasks still get executed [1][2]

[1] [here](https://github.com/openstack-k8s-operators/ci-framework/blob/9526d90e250afaf743f57ce5375da1f12b34e4a3/roles/test_operator/tasks/tempest-tests.yml#L17)
[2] [here](https://github.com/openstack-k8s-operators/ci-framework/blob/9526d90e250afaf743f57ce5375da1f12b34e4a3/roles/test_operator/tasks/tempest-tests.yml#L51)

Reply:
Mistake on my end. This works as expected.